Sources/Orbit/Support/Forms/FormFieldWrapper.swift

```diff
@@ -16,6 +17,9 @@ public struct FormFieldWrapper<Content: View>: View {
 
             content()
 
+            PasswordStrengthIndicator(passwordStrength: passwordStrength)
```

Comment:
The indicator should be provided as a part of `content` from the parent view, not used directly in this generic wrapper.

Reply:
I've moved it mimic the FormFieldMessage 🤷 Will move it back to `InputField`